pkg/apis/pingcap/v1alpha1/tidbmonitor_types.go

```diff
@@ -86,6 +86,26 @@ type PrometheusSpec struct {
 
 	// +optional
 	Ingress *IngressSpec `json:"ingress,omitempty"`
+
+	// +optional
+	Config *PrometheusConfiguration `json:"config,omitempty"`
+}
+
+// +k8s:openapi-gen=true
+// Config  is the the desired state of Prometheus Configuration
+type PrometheusConfiguration struct {
+
+	// user can mount prometheus rule config with external configMap.If use this feature, the external configMap must contain `prometheus-config` key in data.
+	ConfigMapRef *ConfigMapRef `json:"configMapRef,omitempty"`
+
+	// user can  use it specify prometheus command options
+	CommandOptions []string `json:"command,omitempty"`
+}
+
+// ConfigMapRef is the external configMap
+type ConfigMapRef struct {
+	Name      string  `json:"name,omitempty"`
+	Namespace *string `json:"namespace,omitempty"`
```

Comment:
add `// +optional` comment here. And also please add the comment to point out that if the namespace is omited, the operator controller would use the Tidbmonitor's namespace instead.

Reply:
done